src/csharp/Grpc.Examples.MathClient/project.json

```diff
@@ -0,0 +1,32 @@
+{
+  "buildOptions": {
+    "compile": "**/*.cs",
+    "copyToOutput": {
```

Comment:
No, these libraries will be picked up from the Grpc.Core nuget package thanks to the logic in grpc's NativeExtension loader.

These files are only needed for the projects in the gRPC repo to work (for old-style projects this is achieved by a NativeDeps.target files included in our .csproj files, and this is a workaround to make it work). Normally, "copyToOutput" would be only declared for Grpc.Core project, but in CoreCLR, the "copyToOutput" is not transitive so I need to keep repeating myself.

Again, I would have added an explanatory comment, but the project.json files don't support that/